R/motifs.R

```diff
@@ -229,7 +229,7 @@ sample_motifs <- function(graph, size = 3, cut.prob = rep(0, size),
   on.exit(.Call(R_igraph_finalizer))
   .Call(
     R_igraph_motifs_randesu_estimate, graph, as.numeric(size),
-    as.numeric(cut.prob), as.numeric(sample.size), as.numeric(sample)
+    as.numeric(cut.prob), as.numeric(sample.size), sample
```

Comment:
```suggestion
    as.numeric(cut.prob), as.numeric(sample.size), if (!is.null(sample) as_igraph_vs(sample) else NULL
```

I think the correct way to proceed is what I outline above. Can you change this before merging, @krlmlr ?